packages/router/src/components/router.rs

```diff
@@ -28,6 +28,9 @@ pub struct RouterProps<'a> {
     /// This is useful if you don't want to repeat the same `active_class` prop value in every Link.
     /// By default set to `"active"`.
     pub active_class: Option<&'a str>,
+
+    /// Set the active url.
```

Comment:
A bit more detail in the docs would be nice here...

Is this the initial URL? If so, `initial_url` might be a better name. As well as a note on what the default value would be.

Or is it a fixed URL that, when specified, cannot be changed with the push_route functions?

Reply:
I'll update the docs and yes, `initial_url` is a better name.

This is intended to specify the initial url when working specifically with `ssr`, but I'm sure it will be helpful also for mobile and desktop, but I didn't tested those scenarios.